### ISSUE TITLE
Cannot complete quest Return to Kheyna at level 60

### ISSUE THREAD

**Issue by lokhengchau** [slowtorta/turtlewow-bug-tracker#1622]
**Please, provide a clear description what the bug is:**

The quest Return to Kheyna at Tanaris asks you to return to the crystal and talk to Kheyna Spinpistol. I did not see this quest before but now that I'm level 60, when right clicking on the crystal at the crash site (48.1, 23.4), Chromie appears instead of Kheyna. Kheyna actually can be found in Caverns of Time but cannot right click on her and complete the quest either.

**Steps to reproduce the behavior with as much detail as possible:**

1. Reach lvl 60.
2. Pick up quest Return to Kheyna in Tanaris.
3. Cannot complete.

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**

NPC:
Kheyna: https://database.turtle-wow.org/?npc=65004
Reas: https://database.turtle-wow.org/?npc=51266
Quest: https://database.turtle-wow.org/?quest=80409
Object: https://database.turtle-wow.org/?object=3000246

**Expected behavior. Describe how it should work:**
Be able to turn in and complete the quest, either by using the crystal or directly going to Kheyna at Cavern of Time.

**Comment by Gurky-Kronos** [comment]
Happened at level 40 to me as well, at some point this broke as it used to work.

**Comment by usedcars4cheap** [comment]
https://database.turtle-wow.org/?object=3000284 is the object in question - it will spawn Kheyna and allow quest turn-in.

**Comment by madScripting** [comment]
Can confirm, picked the quest there but when trying to turnin the quest, clicking on the object spawns Chromie.
Having another player that did not complete the quest clicking the object will successfully spawn Kheyna Spinpistol and allow you to turnin the quest.

**Comment by redmagejoe** [comment]
While I traveled all the way to Barrens to the first crystal to turn in this quest, I decided to pour over the quest text carefully in-game and on the database, and I see what the problem is with this quest. From reading the context given, it seems that Chromie is supposed to spawn, narratively, but the quest is asking that you turn it into Kheyna. It should instead have you turning it in to Chromie.

This needs to be corrected in the quest database itself. Furthermore, for the sake of addressing all bugs, I would advise making this a separate unique NPC ID of Chromie that doesn't give quests in Andorhal, similar to the convention of making duplicate NPCs in retail to avoid this exact janky and unpolished behavior.

In short, a new Chromie copy should be made with a unique NPC ID whose specific purpose is to be the turn-in for this quest and explains that the Kheyna you've been helping is, in fact, not the Kheyna you encounter in Caverns of Time and has an ulterior motive in seeking the player's help.
